task/buildah/0.3/buildah.yaml

```diff
@@ -93,7 +95,11 @@ spec:
 
   - name: digest-to-results
     image: $(params.BUILDER_IMAGE)
-    script: cat $(workspaces.source.path)/image-digest | tee /tekton/results/IMAGE_DIGEST
+    script: |
+    
+      cat $(workspaces.source.path)/image-digest | tee /tekton/results/IMAGE_DIGEST
```

Comment:
Perhaps change it to tee `$(results.IMAGE_DIGEST.path)` here too for consistency.

Reply:
@simonbaird definitely 😉

Reply:
Done 👍